Dockerfile

```diff
@@ -784,14 +782,16 @@ RUN \
 
 FROM sdk as sdk-cpp
 
-ARG AWS_SDK_CPP_VER="1.9.332"
+ARG AWS_SDK_CPP_VER="1.11.207"
 
 USER builder
 WORKDIR /home/builder/aws-sdk-cpp-src
 COPY ./hashes/aws-sdk-cpp /home/builder/aws-sdk-cpp-src/hashes
 
+# Upstream source fallback is explicitly disabled here as the SHA512 hash
+# verification fails due to a difference in the upstream names and the SDK's.
 RUN \
-  sdk-fetch hashes && \
+  UPSTREAM_SOURCE_FALLBACK=false sdk-fetch hashes && \
```

Comment:
Is the issue here that our files are named `aws-c-sdkutils.tar.gz` instead of `aws-c-sdkutils-a6fd80cf7c163062d31abb28f309e47330fbfc17.tar.gz`?

Reply:
Actually what will end up happening is the file will download as `a6fd80cf7c163062d31abb28f309e47330fbfc17`.